dcpquery/etl/__init__.py

```diff
@@ -110,7 +110,44 @@ def load_bundle(self, bundle, extractor=None, transformer=None):
         config.db_session.commit()
 
 
-def create_view_tables(extractor):
+def update_process_join_table():
+    config.db_session.execute(
+        """
+        Insert into process_join_table (child_process_uuid, parent_process_uuid)
+select * from (
+              with
+                  input_files_table as (
+                                        select
+                                               process_uuid as child_process,
+                                               file_uuid    as input_file_uuid
+                                        from process_file_join_table
+                                        where process_file_connection_type = 'INPUT_ENTITY'
+                                      ),
+
+                  output_files_table as (
+                                          SELECT
+                                                 process_uuid as parent_process,
+                                                 file_uuid as output_file_uuid
+                                        from process_file_join_table
+                                        where process_file_connection_type = 'OUTPUT_ENTITY'
+                                      )
+                  select
+                         input_files_table.child_process,
+                         output_files_table.parent_process
+                  from input_files_table, output_files_table
+       where input_files_table.input_file_uuid = output_files_table.output_file_uuid) as temp_tables
+       ON CONFLICT DO NOTHING;
+        """
+    )
+    config.db_session.commit()
+
+
+def query_service_finalizer(extractor):
```

Comment:
Please rename this to a verb-based name, like `dcpquery_etl_finalize()`.

Reply:
👍